runtime/runtime/src/lib.rs

```diff
@@ -562,7 +565,7 @@ impl Runtime {
 
         // state_update might already have some updates so we need to make sure we commit it before
         // executing the actual receipt
-        state_update.commit();
+        state_update.commit(StateChangeCause::ReceiptProcessing { hash: receipt.get_hash() });
```

Comment:
It should be `ActionReceiptProcessingStarted`, since we didn't process the receipt yet.

Reply:
@nearmax What is your opinion on this?

Reply:
I can see it modifies state_update in the same function, so we cannot assert that the all changes were committed. Let's add this variant, and later revisit it. It is unlikely that current clients will encounter it in their RPC responses, because it is not related to contract state.